unyt/unit_registry.py

```diff
@@ -39,6 +40,15 @@ def _sanitize_unit_system(unit_system, obj):
     return unit_system_registry[str(unit_system)]
 
 
+@lru_cache
```

Comment:
To support Python3.7 and older, this needs to be `@lru_cache()` see https://stackoverflow.com/questions/47218313/use-functools-lru-cache-without-specifying-maxsize-parameter

Reply:
In the rest of unyt, I see `@lru_cache(maxsize=128, typed=False)`. Those are the default parameters, but the fact that they're repeated in the code made me think there was some thought put into the choice. Was there? Should there be other values here?

Reply:
No, I just wanted to make it explicit in case I ever wanted to experiment with changing them.